frontend/src/components/Home/SidePaneView.js

```diff
@@ -3,59 +3,73 @@ import {Split, Stack, StackItem} from "@patternfly/react-core";
 import CardView from "../PatternflyComponents/Card/CardView";
 import React, {useEffect, useState} from "react";
 import {FormSelectView} from "../PatternflyComponents/Form/FormSelectView";
-import { updateDataFilter} from "../../store/reducers/JobsReducer";
-import {fetchJobsData} from "../../store/Actions/ActionCreator";
+import {updateDataFilter} from "../../store/reducers/OCPJobsReducer";
```

Comment:
@chentex Here you need to import the updateDataFilter from CptJobsReducer

Reply:
Great catch @athiruma!! Thank you!!!